Fika.Core/Coop/Factories/PingFactory.cs

```diff
@@ -75,19 +76,16 @@ protected void Update()
                 }
             }
 
-            Camera camera = CameraClass.Instance.Camera;
-
             if (CameraClass.Instance.SSAA != null && CameraClass.Instance.SSAA.isActiveAndEnabled)
             {
                 int outputWidth = CameraClass.Instance.SSAA.GetOutputWidth();
                 float inputWidth = CameraClass.Instance.SSAA.GetInputWidth();
                 screenScale = outputWidth / inputWidth;
             }
 
-            Vector3 screenPoint = camera.WorldToScreenPoint(hitPoint);
-            if (screenPoint.z > 0)
+            if (WorldToScreen.GetScreenPoint(hitPoint, mainPlayer, out Vector3 screenPoint))
             {
-                float distanceToCenter = Vector3.Distance(screenPoint, new Vector3(x: Screen.width, Screen.height, 0) / 2);
+                float distanceToCenter = Vector3.Distance(screenPoint, new Vector3(x: Screen.width, y: Screen.height, 0) / 2);
```

Comment:
We could remove `x:` and `y:`, they must've been left there mistakenly in the old code.